sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -104,18 +107,33 @@ def _run_wordcount_it(self, run_wordcount, **opts):
     run_time = end_time - start_time
 
     if publish_to_bq:
-      bq_publisher = BigQueryMetricsPublisher(
-          project_name=test_pipeline.get_option('project'),
-          table=test_pipeline.get_option('metrics_table'),
-          dataset=test_pipeline.get_option('metrics_dataset'),
-      )
-      result = Metric(
-          submit_timestamp=time.time(),
-          metric_id=uuid.uuid4().hex,
-          value=run_time,
-          label='Python performance test',
-      )
-      bq_publisher.publish([result.as_dict()])
+      self._publish_metrics(test_pipeline, run_time)
+
+  def _publish_metrics(self, pipeline, metric_value):
```

Comment:
Sounds like this code can be reused across other performance tests, is there a module shared across perf tests that we can move it to? cc: @kamilwu

Reply:
Looks like we have  apache_beam.testing.load_tests.load_test_metrics_utils. Should we move it there?

Reply:
I can add something like "publish_single_value(bq/influx/console)" for each publisher

Reply:
`MetricsReader` from `load_test_metrics_utils` module has a `publish_metrics` method. This method takes a pipeline result as an input, extracts metrics from it and calls all publishers. I think we could add a similar method, e.g. `publish_value`, which would take a list of kv pairs. @piotr-szuberski Does it make sense?

Reply:
I think that's a very good idea.